docs/pattern-2.md

```diff
@@ -225,6 +225,9 @@ The pattern exports these outputs to the parent stack:
 
 **Stack Deployment Parameters:**
 - `ClassificationMethod`: Classification methodology to use (options: 'multimodalPageLevelClassification' or 'textbasedHolisticClassification')
+- **OCR**: Control the ability to tune ocr via configuration file `ocr.tuning` property.
```

Comment:
I'm afraid I am confused now by the scope of this PR, including the introduction of this 'tuning' concept.  I had thought this would be a simple PR, a quick win, to introduce only an 'enable' property for classification and extraction, with the simple logic in the associated lambdas to skip the associated processing if enabled is False.   (Like we have already for Summarization).  
And so I am confused by why this would require introduction of new stack deployment parameters and these .tuning properties.